airbyte-integrations/connectors/source-tplcentral/acceptance-test-config.yml

```diff
@@ -0,0 +1,30 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-tplcentral:dev
+tests:
+  spec:
+    - spec_path: "source_tplcentral/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      empty_streams: []
+# TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
+#      expect_records:
+#        path: "integration_tests/expected_records.txt"
+#        extra_fields: no
+#        exact_order: no
+#        extra_records: yes
```

Comment:
this can't be done ?

Reply:
While 3PL Central supports sandbox accounts, sadly, our account tier doesn't include it. In addition, I removed commented parts of the config.